From 6a326662601a16bd586d5b87cc9394220dae4c4b Mon Sep 17 00:00:00 2001
From: Pawel Kot <gnokii@gmail.com>
Date: Sat, 25 Aug 2012 21:03:41 +0000
Subject: Rough cleanup of inline usage.

Make inline to be static inline where relevant (to be compatible with C99).
Remove inline where not necessary.
Convert inline static to static inline.
---
diff --git a/common/gsm-filetypes.c b/common/gsm-filetypes.c
index 053a971..08efa36 100644
--- a/common/gsm-filetypes.c
+++ b/common/gsm-filetypes.c
@@ -1391,7 +1391,7 @@ static int get_next_token(char *src, int delim)
 #define GET_NEXT_TOKEN()	o = get_next_token(line + offset, ';')
 #define STORE_TOKEN(a)		strip_slashes(a, line + offset, sizeof(a) - 1, o - 1)
 
-inline int local_atoi(char *str, int len)
+static int local_atoi(char *str, int len)
 {
 	int retval;
 	char *aux = strndup(str, len);
diff --git a/smsd/lowlevel.c b/smsd/lowlevel.c
index d50f131..d421932 100644
--- a/smsd/lowlevel.c
+++ b/smsd/lowlevel.c
@@ -41,7 +41,7 @@ inline void InsertEvent (PhoneEvent *event)
 }
 
 
-inline static PhoneEvent *RemoveEvent (void)
+static inline PhoneEvent *RemoveEvent (void)
 {
   GSList *list;
   PhoneEvent *event = NULL;
diff --git a/xgnokii/xgnokii.c b/xgnokii/xgnokii.c
index 01a5f25..0948828 100644
--- a/xgnokii/xgnokii.c
+++ b/xgnokii/xgnokii.c
@@ -782,7 +782,7 @@ inline void GUI_ShowAbout(void)
 }
 
 
-inline void GUI_HideAbout(void)
+static inline void GUI_HideAbout(void)
 {
 	gtk_widget_hide(AboutDialog);
 }
diff --git a/xgnokii/xgnokii_logos.c b/xgnokii/xgnokii_logos.c
index 76e690c..c0e94c6 100644
--- a/xgnokii/xgnokii_logos.c
+++ b/xgnokii/xgnokii_logos.c
@@ -1288,7 +1288,7 @@ gint LogoTypeEvent(GtkWidget * widget)
 	return 0;
 }
 
-inline void CloseLogosWindow(void)
+static inline void CloseLogosWindow(void)
 {
 	gtk_widget_hide(GUI_LogosWindow);
 }
--
cgit v0.9.0.2
